config/maintainers.json

```
@@ -70,6 +70,16 @@
       "link_url": "null",
       "avatar_url": null,
       "bio": "I was introduced to this site during Hacktoberfest a few years ago where I helped auto generate some of the solutions for the C# track. I'm professional a C# developer and enjoy working with it"
+    },
+    {
+      "github_username": "nathanchere",
+      "alumnus": false,
+      "show_on_website": true,
```

```suggestion
      "show_on_website": false,
```

Could you set this to `false` for the moment? We'll be setting this to `true` for all contributing maintainers later.